### DISCUSSION TITLE
Error installing.

### DISCUSSION THREAD

**Discussion by cemarta7** [lucky-media/landtamic#8] [General]
I keep getting this error.

```
Validating starter kit license...
License required for [lucky-media/landtamic]!
This is a paid starter kit. If you haven't already, you may purchase a license at:
https://statamic.com/starter-kits/luckymedia/landtamic
```
I never have the option to input the license key.

**Comment by cemarta7** [comment]
It worked doing statamic new my-site lucky-media/landtamic

**Reply by lokmanm** [reply]
Hi @cemarta7, based on the error that you were getting it was a license issue. 

Maybe you didn't paste the license on the terminal ? 

Never the less I am glad you got it resolved.

**Comment by malle-pietje** [comment]
I am having the same issue here. Building a test environment here with Statamic and trying to install a Landtamic license I just purchased. Getting this error:
```
$ php please starter-kit:install lucky-media/landtamic
Validating starter kit license...
License required for [lucky-media/landtamic]!
This is a paid starter kit. If you haven't already, you may purchase a license at:
https://statamic.com/starter-kits/luckymedia/landtamic
```
When I create a new site I get this:
```
$ statamic new landtamic-test lucky-media/landtamic

   _____ __        __                  _
  / ___// /_____ _/ /_____ _____ ___  (_)____
  \__ \/ __/ __ `/ __/ __ `/ __ `__ \/ / ___/
 ___/ / /_/ /_/ / /_/ /_/ / / / / / / / /__
/____/\__/\__,_/\__/\__,_/_/ /_/ /_/_/\___/


This is a paid starter kit. If you haven't already, you may purchase a license at:
https://statamic.com/starter-kits/luckymedia/landtamic
Please enter your license key: zzzzzzzzzzzzzzzzzzzzz
Starter kit license valid!

Once successfully installed, this single-site license will be marked as used
and cannot be installed on future Statamic sites! Would you like to continue installation?
I am aware this is a single-site license (yes/no) [no]: no
```
But that's not what I want, I simply wish to add Landtamic to the current test application that we've been working on.

Do I need to add an entry to the .env file with the license key or something else?

**Comment by malle-pietje** [comment]
Arghhh, hopefully nobody saw that license key...

**Comment by malle-pietje** [comment]
This is taking quite a bit of my time already just so that I can test whether Landtamic meets my requirements....

**Comment by lokmanm** [comment]
@malle-pietje it looks like you have licensing issues.
It's best to contact Statamic Support as we don't have access to that.

**Reply by malle-pietje** [reply]
Okay, will do.

**Comment by malle-pietje** [comment]
For others with this same issue; you need to tie the Starter-pack license to a Statamic site through your account, the apply the Statamic license key to your `.env` file.
